poojasanwal.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Pooja Sanwal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "She/Her, I talk about development, technical writing, open source and communities",
+  "avatar": "https://github.com/poojasanwal.png",
```

Comment:
```suggestion
  "type": "personal",
  "bio": "She/Her, I talk about development, technical writing, open source and communities",
```

Reply:
We don't need the `displayStatsPublic` and `avatar` after the recent update, So remove these 2 lines.

Reply:
Thanks for reviewing. I have removed the two lines.

Reply:
> Thanks for reviewing. I have removed the two lines.

ya, but move it under `data` folder

Reply:
> > Thanks for reviewing. I have removed the two lines.
> 
> ya, but also move it under the `data` folder